src/coreclr/src/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -8,16 +8,21 @@ internal static partial class GlobalizationMode
     {
         internal static bool Invariant { get; } = GetGlobalizationInvariantMode();
 
-        // GetInvariantSwitchValue calls CLRConfig first to detect if the switch is defined in the config file.
+        internal static bool GetInvariantSwitchValue() =>
+            GetSwitchValue("System.Globalization.Invariant", "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT");
+
+        internal static bool GetUseNlsSwitchValue() =>
+            GetSwitchValue("System.Globalization.UseNls", "DOTNET_SYSTEM_GLOBALIZATION_USENLS");
```

Comment:
>DOTNET_SYSTEM_GLOBALIZATION_USENLS [](start = 59, length = 34)

the term NLS could be not popular. I suggest to name this UseWin32

Reply:
We should stick with established names and invent our own. I do not see Win32 in any of the docs talking about NLS vs ICU differences.

In fact, ICU globalization APIs are under Win32 path in the docs [https://docs.microsoft.com/en-us/windows/win32/intl/international-components-for-unicode--icu-](https://docs.microsoft.com/en-us/windows/win32/intl/international-components-for-unicode--icu-)

Reply:
Looking one more time at the docs, I am seeing the NLS is well defined. I agree then staying with NLS 

[https://docs.microsoft.com/en-us/windows/win32/intl/national-language-support](https://docs.microsoft.com/en-us/windows/win32/intl/national-language-support)

---
In reply to: [405674417](https://github.com/dotnet/runtime/pull/34645#discussion_r405674417) [](ancestors = 405674417)